File: java/JavaFileutil/src/JavaFileutil/Validators/NonAbsolutePathValidator.java
package JavaFileutil.Validators;

import java.util.List;

public class NonAbsolutePathValidator extends AbstractFilesystemEntryValidator
{
    public NonAbsolutePathValidator(List<String> args, short index, String message)
    {
        super(args, index, message);
    }

    @Override
    public boolean Validate()
    {
        if(file == null || file.isAbsolute())
        {
            System.out.println(ErrorPreface + Message);
            return false;
        }

        return true;
    }
}

File: python/Validators/NonAbsolutePathValidator.py
from .AbsolutePathValidator import *


class NonAbsolutePathValidator(AbsolutePathValidator):
    def __init__(self, validate_this_path, message):
        super().__init__(validate_this_path, message)

    def validate(self):
        if self._validate_this.is_absolute():
            self._print_validation_error()
            return False
        return True
